lib/tasks/migrate.rake

```diff
@@ -11,17 +11,41 @@ namespace :migrate do
     end
   end
 
-  desc "Migrated Ephemera Folders in Ephemera Boxes published in production to a completed workflow state"
+  desc "Migrates Ephemera Folders in Ephemera Boxes published in production to a completed workflow state"
   task ephemera_folders: :environment do
     # Ensures that all member EphemeraFolders have their state properly updated
-    resources.each do |resource|
+    resources(model: EphemeraFolder).each do |resource|
       cs = DynamicChangeSet.new(resource)
       cs.prepopulate!
       logger.info "Migrating folders within the box #{resource.id}..."
       change_set_persister.save(change_set: cs)
     end
   end
 
+  desc "Migrates Collection members with children who have values in the member_of_collection_ids attribute"
+  task collection_members_with_children: :environment do
+    # Ensures that all member EphemeraFolders have their state properly updated
+    resources(model: Collection).each do |collection|
+      logger.info "Migrating Collection members for #{collection.id}..."
+
+      change_set_persister.buffer_into_index do |buffered_change_set_persister|
+        collection.decorate.decorated_members.each do |collection_member|
+          collection_member.members.each do |child|
+            next if child.member_of_collection_ids.empty?
+
+            logger.info "Migrating the collections for member resource #{child.id}..."
+
+            child_change_set = DynamicChangeSet.new(child)
+            child_change_set.prepopulate!
+            child_change_set.member_of_collection_ids = []
```

Comment:
Did you test this out? I would have expected it to be more like `child_change_set.validate(member_of_collection_ids: [])`